solr/core/src/java/org/apache/solr/search/FunctionRangeQuery.java

```diff
@@ -42,7 +43,12 @@ public FunctionRangeQuery(ValueSourceRangeFilter filter) {
   @Override
   public DelegatingCollector getFilterCollector(IndexSearcher searcher) {
     Map<Object,Object> fcontext = ValueSource.newContext(searcher);
-    Weight weight = rangeFilt.createWeight(searcher, ScoreMode.COMPLETE, 1);
+    Weight weight = null;
+    try {
+      weight = rangeFilt.createWeight(searcher, ScoreMode.COMPLETE, 1);
+    } catch (IOException e) {
+        throw new SolrException(SolrException.ErrorCode.SERVER_ERROR, e);
+    }
```

Comment:
nit: indentation here seems inconsistent.

Reply:
Thanks for catching this, fixed!